acbuild/write.go

```diff
@@ -33,7 +34,8 @@ var (
 func init() {
 	cmdAcbuild.AddCommand(cmdWrite)
 
-	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite output ACI")
+	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite the resulting ACI")
+	cmdWrite.Flags().BoolVar(&compress, "compress", false, "compress the resulting ACI")
```

Comment:
Same as earlier I suppose, I think it would make more sense to default to compressing the ACI.